### ISSUE TITLE
I want to contribute.

### ISSUE THREAD

**Issue by eve-moo** [evemuproject/evemu_server#142]
I've looked at this project a few times over the last couple years and since I have some time I though I'd contribute to the project.  I've already made a few changes that I've pushed to my fork at
https://github.com/eve-moo/evemu_server
1) A summary of what I've changed is removing LUA... It's never been used for anything so why is it even there?
2) Removed doxygen... This one may be controversial if anyone is actually using the doxygen files but since they are just html formated versions of the function comments.  It also takes a long time to compile if you do a clean/build.
3) Fixed the compile warnings in EvilNumberTest.cpp
4) Added to the .gitignore file to prevent adding generated files and compiled executables.
I've also started working on some actual productive documentation.  I've worked on the docs as a way of documenting what I've learned over the last couple years. I'll add it to my project as soon as I'm not embarrassed by its quality.  Should be a day or two.
Also, sad to see the web site is gone and IRC seams to have died.  If there is anything I can do to help create a new/better way of communicating other than here please let me know.

**Comment by comet0** [comment]
My thoughts on your changes:
1) Agree. If it does nothing and is being used by nobody why have it l.
2) I wouldn't remove it, I would say just move it to a non main build target so it is not built when only building the server.
3) useful but it is in the testing directory. Not 100% sure we are really bothered by test warnings.
4) can't comment, if you build in root then you probably want those changes. 

All in all, good job, the more the merrier

**Comment by eve-moo** [comment]
I'm using netbeans and when it does a clean and build on a cmake project it rebuilds EVERYTHING.  So, the doxygen does get a little annoying.  As do the warnings in the test apps.  It also by default builds everything in the development directories hence the additions to gitignore.
I've also uploaded the documentation I created.  Let me know what you think, particularly the setup instructions.  I wanna make sure I have that right!

**Comment by VizanArkonin** [comment]
1. LUA was planned as a scripting module to work around NPC AI and probably some event triggers. Frame is there, which, as you can see, was never used. We'll see if we'll need it sometime soon, but for now - yep, we don't use it at all.
2. But why remove it? By default, it is never built, unles you include it in build parameters.
3,4 - As for EvilNumber and .gitignore - not much to comment, as i'm not really into module manager.

Even though allan's mostly away (heavy-duty work schedule), i think he can still accept the pull requests and leave the notes/explanations on the code, as he's been coding the server for quite a while, so if you have something coded - don't hesitate to push it.

Cheers, mate!

**Comment by eve-moo** [comment]
Actually, every time I've tried out evemu in the last 2 years it has downloaded and compiled lua... I've used eclipse and netbeans as IDEs as well as going straight command prompt.  Why remove it?  To clean up the project a bit and it can always be easily re-added if it is needed.
I've put a couple more commits to my fork and created a pull request.
Also, I'd like some comments on the documentation I've been working on.  Would be nice to know if people like it or think it could be better before I create a lot more.
